package.json

```diff
@@ -18,7 +18,7 @@
     "react-dom": "latest",
```

Comment:
Might as well set this and `react` to a version as well

Reply:
set react and react-dom to specific versions